src/Source.ruleset

```diff
@@ -4,16 +4,17 @@
     <Name Resource="MinimumRecommendedRules_Name" />
     <Description Resource="MinimumRecommendedRules_Description" />
   </Localization>
+  <Include Path="Sdl.Required.Warning.ruleset" Action="Default" />
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
-    <Rule Id="CA1001" Action="Warning" />
+    <!-- <Rule Id="CA1001" Action="Warning" /> Disable. Don't Fix -->
     <Rule Id="CA1009" Action="Warning" />
     <Rule Id="CA1016" Action="Warning" />
-    <Rule Id="CA1033" Action="Warning" />
+    <!-- <Rule Id="CA1033" Action="Warning" /> Disable. Don't Fix -->
     <Rule Id="CA1049" Action="Warning" />
-    <Rule Id="CA1060" Action="Warning" />
+    <!-- <Rule Id="CA1060" Action="Warning" /> Disable. Might Fix -->
     <Rule Id="CA1061" Action="Warning" />
-    <Rule Id="CA1063" Action="Warning" />
-    <Rule Id="CA1065" Action="Warning" />
+    <!-- <Rule Id="CA1063" Action="Warning" /> Disable. Don't Fix -->
+    <!-- <Rule Id="CA1065" Action="Warning" /> Disable. Might Fix -->
```

Comment:
The rules that are disabled on this file correspond to what is described on #5334